src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsDrawer/ItemSettingsDrawer.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent } from '@testing-library/react';
+
+import { renderWithProviders } from 'shared/utils';
+
+import { ItemSettingsDrawer } from './ItemSettingsDrawer';
+
+const dataTestid = 'builder-activity-items-item';
+
+const onClose = jest.fn();
+const props = {
+  open: true,
+  onClose,
+};
+
+describe('ItemSettingsDrawer Component', () => {
+  test('renders component with correct title, icons and content when open, calls onClose callback when cross is clicked', () => {
+    const { getByTestId, getByText } = renderWithProviders(
```

Comment:
it's better using these methods from 'screen' imported from '@testing-library/react'